docs/source/acknowledgements.rst

```diff
@@ -8,7 +8,8 @@ warehouses, and deployment logistics into a federated 'data distribution'"
 1429999`_) and the German Federal Ministry of Education and Research (`BMBF
 01GQ1411`_). Additional support is provided by the German federal state of
 Saxony-Anhalt and the European Regional Development
-Fund (ERDF), Project: `Center for Behavioral Brain Sciences`_, Imaging Platform
+Fund (ERDF), Project: `Center for Behavioral Brain Sciences`_, Imaging Platform.
+DataLad also received support by the Deutsche Forschungsgemeinschaft (DFG, German Research Foundation) under grants `SFB 1451 (431549029, INF project)`_ and `CRC 379 (512007073, Q02)`_.
```

Comment:
```suggestion
DataLad also received support by the Deutsche Forschungsgemeinschaft (DFG,
German Research Foundation) under grants `SFB 1451 (431549029, INF project)`_
and `CRC 379 (512007073, Q02)`_.
```

for consistency with the rest of the text.

Otherwise good, and for codespell new gotchas I sent

- #7698